File: PYTHON/Program/Shirt.py
from Product import Product
from Clothing import Clothing

class Shirt(Clothing):
    __color = ""
    __sleeveType = ""

    def __init__(self):
        self.__color = ""
        self.__sleeveType = ""

    def setColor(self, color):
        self.__color = color

    def getColor(self):
        return self.__color
    
    def setSleeveType(self, sleeveType):
        self.__sleeveType = sleeveType

    def getSleeveType(self):
        return self.__sleeveType

File: PHP/Program/Shirt.php
<?php

class Shirt extends Clothing{
    private $color;
    private $sleeveType;

    function __construct(){
    }

    function setColor($color){
        $this->color = $color;
    }

    function getColor(){
        return $this->color;
    }

    function setSleeveType($sleeveType){
        $this->sleeveType = $sleeveType;
    }

    function getSleeveType(){
        return $this->sleeveType;
    }

    function __destruct(){
    }
}

?>